corehq/form_processor/submission_post.py

```diff
@@ -192,7 +192,9 @@ def run(self):
                     else:
                         instance.initial_processing_complete = True
                         self.save_processed_models(xforms, case_stock_result)
-                        case_stock_result.case_result.close_extensions(case_db)
+                        device_id = instance.metadata.deviceID if instance.metadata else ""
+                        case_stock_result.case_result.close_extensions(case_db,
+                            "SubmissionPost-close_extensions-%s" % device_id)
```

Comment:
I'm not suggesting you add this to the device ID but it would be great to know in the form which triggered this when looking at the case close form.